content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -332,14 +332,18 @@ container of a Pod can specify either or both of the following:
 
 Limits and requests for `ephemeral-storage` are measured in byte quantities.
 You can express storage as a plain integer or as a fixed-point number using one of these suffixes:
-E, P, T, G, M, K. You can also use the power-of-two equivalents: Ei, Pi, Ti, Gi,
+E, P, T, G, M, k. You can also use the power-of-two equivalents: Ei, Pi, Ti, Gi,
 Mi, Ki. For example, the following quantities all represent roughly the same value:
 
 - `128974848`
 - `129e6`
 - `129M`
 - `123Mi`
 
+Pay attention to the case of the suffixes. If you request `400m` of ephemeral-storage, this is a request
+for 0.4 bytes. Someone who types that probably meant to ask for 400 mebibytes (`400Mi`)
```

Comment:
When you specify a quantity of ephemeral storage, pay attention to the case of the suffix. For example, `400m` of ephemeral-storage is equal to 0.4 bytes, not 400 mebibytes (`400Mi`) or 400 megabytes (`400M`).

Reply:
The wording that's I committed is basically an exact copy of what currently exists for the [same warning for `memory`.](https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/configuration/manage-resources-containers.md?plain=1#L136-L138)

I can change the wording of this, but would be curious if you'd additionally prefer that wording for `memory` should also be  changed to this new phrasing.  (I think your wording may be better)

Reply:
Good question. It is okay to leave the wording as is.